src/Raven.Server/Documents/Handlers/BatchHandler.cs

```diff
@@ -241,6 +241,12 @@ public override void Execute(DocumentsOperationContext context)
                             cmd.Document.TryGet(Constants.Documents.Metadata.Key, out metadata);
                             LastEtag = putResult.Etag;
 
+                        
+                            metadata.Modifications = new DynamicJsonValue(metadata)
```

Comment:
Technically speaking, we can just keep the metadata fields, and read the values from there on the client, I guess.